### ISSUE TITLE
Clock uses universal time, not local time

### ISSUE THREAD

**Issue by vonostrowski** [mathoudebine/homeassistant-timebox-mini#18]
Hi.
Thanks for this integration that gave my timebox mini a new purpose. Everthing works with Home Assistant (2024.6.3) installed in docker on a Raspberry Pi 3 - except one thing: The clock shows the universal time (that I see in sudo raspi-config), but not the local time, which is set correct both in raspi-config and also in the home assistant settings. I have no idea how I could fix that. Any idea?

**Comment by mathoudebine** [comment]
Hello, thanks for raising this issue.
Just to be sure, did you use the service with the `set_time` action to set the Timebox clock?

**Comment by vonostrowski** [comment]
Yes. I set the time via the set_time action and then used the clock-action to show the time.
